src/telliot_feeds/utils/decode.py

```diff
@@ -0,0 +1,108 @@
+"""
+Helper functions for decoding query data and submitted values.
+"""
+from typing import Any
+from typing import Callable
+from typing import Optional
+
+import eth_abi
+from telliot_core.utils.response import error_status
+from telliot_core.utils.response import ResponseStatus
+
+from telliot_feeds.queries.abi_query import AbiQuery
+from telliot_feeds.queries.json_query import JsonQuery
+from telliot_feeds.queries.legacy_query import LegacyRequest
+from telliot_feeds.queries.query import OracleQuery
+from telliot_feeds.queries.query_catalog import query_catalog
+
+
+def bytes_from_string(
```

Comment:
I don't think you need this function. you can just use this library: ```from eth_utils.conversions import to_bytes```, then ```to_bytes(hexstr="0x123")``` (0x is optional)  and would that take care of the 0x problem.